stable/atlantis/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "v0.7.1"
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 3.3.1
+version: 3.6.3
```

Comment:
A bit of an odd diff ... but the version in helm:master is 3.6.2 

https://github.com/helm/charts/blob/2f0cfa3bc222651811955cbc9f43d0fd4d130038/stable/atlantis/Chart.yaml#L5